src/base/interfaces/istringable.h

```diff
@@ -35,6 +35,5 @@ class IStringable
 public:
     virtual ~IStringable() = default;
 
-    // requirement: T(const QString &) constructor for derived class `T`  // TODO: try enforce it in C++20 concept
```

Comment:
You should only remove the TODO part. Also I'm still evaluating this so I would keep it as is a little longer.

Reply:
>The constructor is not part of the interface. We cannot and should not require it here. 

FYI, I was thinking of [C++ Named Requirements](https://en.cppreference.com/w/cpp/named_req) when I implemented it.